testdata/baselines/reference/submoduleAccepted/conformance/checkJsdocSatisfiesTag9.types.diff

```diff
@@ -0,0 +1,10 @@
+--- old.checkJsdocSatisfiesTag9.types
++++ new.checkJsdocSatisfiesTag9.types
+@@= skipped -12, +12 lines =@@
+ >Palette : { white: { r: number; g: number; b: number; }; black: { r: number; g: number; d: number; }; blue: { r: number; g: number; b: number; }; }
+ >({    white: { r: 255, g: 255, b: 255 },    black: { r: 0, g: 0, d: 0 }, // <- oops! 'd' in place of 'b'    blue: { r: 0, g: 0, b: 255 },}) : { white: { r: number; g: number; b: number; }; black: { r: number; g: number; d: number; }; blue: { r: number; g: number; b: number; }; }
+ >{    white: { r: 255, g: 255, b: 255 },    black: { r: 0, g: 0, d: 0 }, // <- oops! 'd' in place of 'b'    blue: { r: 0, g: 0, b: 255 },} : { white: { r: number; g: number; b: number; }; black: { r: number; g: number; d: number; }; blue: { r: number; g: number; b: number; }; }
++>{    white: { r: 255, g: 255, b: 255 },    black: { r: 0, g: 0, d: 0 }, // <- oops! 'd' in place of 'b'    blue: { r: 0, g: 0, b: 255 },} : { white: { r: number; g: number; b: number; }; black: { r: number; g: number; d: number; }; blue: { r: number; g: number; b: number; }; }
```

Comment:
Some of these baselines only differ because they have an extra duplicate line. Is this just something that could be skipped in the type baseliner?

Reply:
probably. right now it intentionally does not skip as/satisfies, but I can see if it's possible to skip the node itself but not its children

Reply:
ok, done. the code is a little goofy but the baseline diffs are all gone.